config/default/kustomization.yaml

```diff
@@ -16,6 +16,11 @@ namePrefix: platform-operators-
 commonAnnotations:
   release.openshift.io/feature-gate: TechPreviewNoUpgrade
 
+images:
+- name: quay.io/brancz/kube-rbac-proxy
+  newName: gcr.io/kubebuilder/kube-rbac-proxy
+  newTag: v0.8.0
```

Comment:
Can we also do this for RukPak's image?

Reply:
For the `newTag: v0.8.0`, what's the naming rule? Or what's the rule to align with the OCP version?